release-team/role-handbooks/bug-triage/README.md

```diff
@@ -55,233 +33,208 @@ Bug Triage has a lower time requirement than most roles at the beginning of the
+You should use this time to familiarize yourself with the _major enhancements and fixes_ planned by each SIG for this release, so that you can have context in advance of when you will need to identify incoming bugs as being associated with a work focus in the current release. This can greatly help in focused communication with the relevant SIG leads, as in later stages there can be an elevated urgency to fix related critical bugs and deliver new features in time.
 
 It is also a good time to interact with the Enhancements Lead and CI Signal Lead to understand any early concerns they might have, as the release team's risk management comes as much from this proactive collaboration more as from the Bug Triage lead reacting to incoming issues and PRs.
 
+The major and potentially release-blocking PRs/issues in this stage should be identified and a strategy to distribute them among the team should be devised. It is also important to identify _priority/critical-urgent_ and _priority/important-soon_ PRs/issues early on which could become release blocking later in the cycle.
+
 This is the best stage to get involved with any automation work that can ease the workload of later stages. Some tasks include:
 - updating/extending scripts to populate spreadsheets with relevant issues/PRs, delegate them to Bug Triage team members, and track them.
 - automation of notifications on relevant issues/PRs of the release cycle timeline.
 - automation of tracking/categorizing issues/PRs by responses to notifications of release cycle timeline.
 
-The major PRs/issues in this stage should be identified and a strategy to distribute the workload amongst the team should be devised. A [tracking spreadsheet](https://docs.google.com/spreadsheets/d/14DzSChatdcwKLBxshMkE9ul4jaocBJGPdR1VnrMCqPs) is in use for this purpose and should be populated using the current status of PRs/issues. It is also important to identify _priority/critical-urgent_ and _priority/important-soon_ PRs/issues early on which could become release blocking later in the cycle.
+#### Setting up the Bug Triage Spreadsheet
 
-### Sample Searches [Early]
-* Issues which could require early intervention
-  - [Issues in the v1.17 milestone which haven't been updated in a while](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3Av1.17+updated%3A%3C2019-09-01+repo%3Akubernetes%2Fkubernetes+is%3Aopen+): `is:issue milestone:v1.17 updated:<2019-09-01 repo:kubernetes/kubernetes is:open`
+The Bug Triage team is using a [tracking spreadsheet](https://docs.google.com/spreadsheets/d/14DzSChatdcwKLBxshMkE9ul4jaocBJGPdR1VnrMCqPs) to track the current status of all issues and PRs targeting the release, their priorities, and to distribute the work among the Bug Triage team. 
```

Comment:
not sure we want to link this sheet, since it's 1.16 specific.

Having sample issue searches in the handbook is really helpful IMO

Reply:
Removed the link.